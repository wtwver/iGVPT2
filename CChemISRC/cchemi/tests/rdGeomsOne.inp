2
Geometry
5 0 1
 Al Al Al Al 0 0.000000000000 2 1 1.374765634553 2.136211928419 -2.077530099581 0 
 O O O Al 0 0.000000000000 2 1 0.000000000000 0.000000000000 0.000000000000 0 
 Al Al Al Al 0 0.000000000000 2 1 0.409155168253 1.890576487181 0.193518496405 0 
 O O O Al 0 0.000000000000 2 1 0.487443665626 0.172026822806 -1.309682957895 0 
 O O O Al 0 0.000000000000 2 1 -0.215727176061 -1.114644821651 0.832762098201 0 
