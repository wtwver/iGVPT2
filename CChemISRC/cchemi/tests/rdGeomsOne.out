-1.04421727229369e+00
1.75723484076405e+00	  2.82665466723948e+00   -2.03995324616005e+00
Geometry
5 0 1
Al Al Al Al 0 0.00000000000000e+00 2 1 1.60443539540109e+00 2.53636664489673e+00 -1.85748373616414e+00 0 
O O O Al 0 0.00000000000000e+00 2 1 -1.29237932203606e+00 4.79250981916145e-01 7.61430080914313e-01 0 
Al Al Al Al 0 0.00000000000000e+00 2 1 1.00872363263303e-01 1.17763646532563e-01 -1.12015153922662e-01 0 
O O O Al 0 0.00000000000000e+00 2 1 8.88471304891456e-01 1.38468037336484e+00 -1.02632952583562e+00 0 
O O O Al 0 0.00000000000000e+00 2 1 7.54237370900909e-01 -1.43389136974136e+00 -1.26534548144917e-01 0 
